tests/e2e/util.go

```diff
@@ -5260,6 +5260,22 @@ func waitForVolumeSnapshotContentToBeDeleted(client snapclient.Clientset, ctx co
 	return waitErr
 }
 
+// waitForCNSSnapshotToBeDeleted wait till the give snapshot is deleted from CNS
+func waitForCNSSnapshotToBeDeleted(volumeId string, snapshotId string) error {
+	var err error
+	waitErr := wait.PollImmediate(poll, pollTimeout, func() (bool, error) {
+		err = verifySnapshotIsDeletedInCNS(volumeId, snapshotId)
+		if err != nil {
+			if strings.Contains(err.Error(), "snapshot entry is still present") {
+				return false, nil
+			}
+			return false, err
+		}
+		return true, nil
```

Comment:
Please add a log statement here saying that snapshot is successfully deleted.

Reply:
Done